### ISSUE TITLE
Active Element after successful delete operation

### ISSUE THREAD

**Issue by hiteshah185** [TeraServ/starfish#80]
Keyboard Focus should be placed on the top[if search box, then dropdown] of the list/table page after **Successful delete** operation. 

- [ ] Focus Styling
